test/integration/setup.js

```diff
@@ -8,5 +8,6 @@ const requiredEnvVars = [
 const undefinedEnvVar = (key) => typeof process.env[key] === 'undefined';
 
 if (requiredEnvVars.some(undefinedEnvVar)) {
-  throw new Error('Please define all necessary "CONTENTFUL_INTEGRATION_*" environment variables');
+  console.error('Please define all necessary "CONTENTFUL_INTEGRATION_*" environment variables');
```

Comment:
Just throwing errors was not yielding the actual message to me, I'd attribute it to mocha :)